discovery-marathon-api/src/main/scala/akka/discovery/marathon/MarathonApiSimpleServiceDiscovery.scala

```diff
@@ -84,6 +84,11 @@ class MarathonApiSimpleServiceDiscovery(system: ActorSystem) extends SimpleServi
 
     log.info("Requesting seed nodes by: {}", request.uri)
 
+    val portName = lookup.portName match {
```

Comment:
Is this variable used anywhere?

Reply:
My bad, I commit the change with the use in targets

Reply:
Aah, that makes a little more sense. Shouldn't it also be used while constructing the query somehow, though?

Reply:
you can't filter by portname in the marathon Rest API

Reply:
you can't filter by portname in the marathon Rest API